modules/selector.js

```diff
@@ -0,0 +1,42 @@
+export const bookList = document.querySelector('#list');
+export const addBook = document.querySelector('#add-new');
+export const contact = document.querySelector('#contact');
+export const addSection = document.querySelector('#adds');
+export const contactSection = document.querySelector('#contacts');
+export const displayArea = document.querySelector('#lists');
+
+export const onload = () => {
+  contactSection.style.display = 'none';
+  // addSection.style.display = 'none';
+  displayArea.style.display = 'block';
+  bookList.classList.add('active');
+  contact.classList.remove('active');
+  addBook.classList.remove('active');
+  // header.style.display = 'block';
+};
+
+// window.addEventListener('load', onload);
+// window.addEventListener('load', displayDate);
+// bookList.addEventListener('click', onload);
+
+// export const addBook.addEventListener('click', () => {
```

Comment:
- [ ] Commented codes render your codebase messy. Do not commit them to your repo